packages/sidecar/src/main.ts

```diff
@@ -97,16 +104,15 @@ function startServer(options: StartServerOptions): ServerType {
     }
   }
 
-  function handleServerListen(port: number, basePath?: string): void {
-    logger.info(`Sidecar listening on ${port}`);
-    if (basePath) {
-      logSpotlightUrl(port);
-    }
+  if (options.stdioMCP) {
```

Comment:
if you're destructuring `port` and `basePath` up there, shouldn't we do the same for `stdioMCP`?